src/contexts/BrowserWorker.js

```diff
@@ -275,6 +278,17 @@ function workerBoilerScript() {
 				return def;
 			}
 			function reject(r, transfers) {
+				if (r instanceof Error) {
+					postMessage({
+						cmd: 'deferred_reject_error',
+						token: data.token,
+						error: {
```

Comment:
If we replace this object literal with `JSON.parse(JSON.stringify(r))` then we can side-step any structured cloning errors while still having a fairly complete view of the error.